src/proj.rs

```diff
@@ -933,6 +989,18 @@ mod test {
             "proj=longlat datum=WGS84 no_defs ellps=WGS84 towgs84=0,0,0"
         );
     }
+
+    #[test]
+    fn test_debug() {
+        let wgs84 = "+proj=longlat +datum=WGS84 +no_defs";
+        let proj = Proj::new(wgs84).unwrap();
+        let debug_string = format!("{:?}", proj);
+        assert_eq!(
+            "Proj { id: Some(\"longlat\"), description: Some(\"PROJ-based coordinate operation\"), definition: Some(\"proj=longlat datum=WGS84 no_defs ellps=WGS84 towgs84=0,0,0\"), has_inverse: true, accuracy: -1.0 }",
```

Comment:
Note that if PROJ changes any of the text here in across PROJ versions, this test will fail. Not sure how concerned we should be about that.